### PR TITLE
Changes subject area (taxonomy) view to match landing page

### PR DESCRIPTION
**Pull request by emb03**
commit bdd5acfb26f0054e538a1af282b085275bd5a57b
Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
Date:   Thu May 19 09:56:42 2016 -0700

    adds css for taxonomy section to match landing page, changes pagination, new version number

commit 18cdd9a1dad7ac4414e0062894e748138ba6f626
Author: Elaine Blakeman <elaineblakeman@library.ucla.edu>
Date:   Thu May 19 08:23:21 2016 -0700

    removes custom templates and reverts to using default templates

### CODE REVIEW

**Comment by emb03** [comment]
@z3cka Can you review and merge to test? Many thanks!

**Comment by z3cka** [comment]
deploying now

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module [hunk 1]
```diff
@@ -168,7 +150,7 @@ function uclalib_bookplates_preprocess_views_view_field(&$variables) {
   if ($variables['field']->field == 'field_collection_donor') {
     // create array of donor names
     $variables['donor_names'] = array();
-    $variables['output_donors'] = '';
+    $variables['output'] = '';
 
     foreach ($variables['row']->field_field_collection_donor as $key => $value) {
       $variables['donor_names'][] = $donor_name;
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module]: @emb03 looks like `$donor_name` has not been set yet here.

Comment (emb03) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module]: it seems to work okay. Can you be more specific?

Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module]: `Notice: Undefined variable: donor_name in uclalib_bookplates_preprocess_views_view_field() (line 156 of /var/www/vhosts/webhead/drupal/www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module).`

Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module]: on line 156 `$donor_name` is not defined yet _(it does not have a value)_. it should be `$value` or change `$value` on line 155 to `$donor_name`. That's all.

**Pull request merged**